wear/src/main/java/io/homeassistant/companion/android/home/HomePresenterImpl.kt

```diff
@@ -106,4 +107,13 @@ class HomePresenterImpl @Inject constructor(
     override suspend fun setWearHomeFavorites(favorites: List<String>) {
         integrationUseCase.setWearHomeFavorites(favorites.toSet())
     }
+
+    override suspend fun getTileShortcuts(): List<SimplifiedEntity> {
+        return integrationUseCase.getTileShortcuts().map { SimplifiedEntity(it) }
+    }
+
+    override suspend fun setTileShortcuts(entities: List<SimplifiedEntity>) {
+        integrationUseCase.setTileShortcuts(entities.map { it.entityString })
+        view.refreshShortcutsTile()
```

Comment:
Rather than pass this around (View -> onClick -> ViewModel -> Presenter -> View) we should just put the call right in the onClick handlers where needed I think?

Reply:
Passing on the view now, to call it immediately in the onClick handler